Add FAQ page (#949)
EuroPython/website: src/content/pages/faq.mdx

---
title: FAQ
subtitle: Frequently Asked Questions about EuroPython
---

# FAQ

## Milestones

> <ul  className="milestone-done">

Author: ```suggestion
<ul className="milestone-done">
```

>   <li>**12 December 2024 Call for Teams starts</li>

Author: ```suggestion
  <li>**12 December 2024** [Call for Contributors](https://forms.gle/kY7jqSJTjEhFccWJ6) starts</li>
```
That's the name on the form.

Reviewer: I don't want to link to the form as the form is still reachable but the Call is through for now.

Author: Ah OK! Understood.

  <li>**10 January 2025** [Call for Proposals (CfP)](/programme/cfp/) opens</li>
  <li>**11 January 2025** [Speaker Mentorship Programme](/programme/mentorship/) sign-up opens</li>
</ul>

<ul className="milestone-todo">
  <li>**21 January 2025:** Speaker Mentorship signup closes</li>
  <li>**27 January 2025:** Call for Proposals (CfP) closes</li>
  <li>**January/February** Financial Aid Programme starts</li>
  <li>**January/February 2025** Community Voting and Proposal Review</li>
  <li>**February/March 2025** Programme selection process</li>

>   <li>**March 2025** Ticket sales open!**</li>

Author: ```suggestion
  <li>**March 2025** Ticket sales open!</li>
```

  <li>**March 2025** Visa Information Page opens</li>
  <li>**01 April 2025** Session preview</li>

>   <li>**20 April 2025** Schedule published</li>

Author: ```suggestion
```

Author: oops, sorry. Ignore this. The suggestion was due to some random idiocy of mine!

>   <li>**??? 2025** Financial Aid Programme closes</li>

Author: ```suggestion
  <li>**TBC 2025** Financial Aid Programme closes</li>
```
??? is giving me build error. So wondering if it is simpler to replace it with TBC

>   <li>**14–20 July 2025:** EuroPython 2025</li>

Author: ```suggestion
  <li>**14–20 July 2025:** EuroPython 2025!🎊 </li>
```
why so serious?

---
title: FAQ
subtitle: Frequently Asked Questions about EuroPython
---

# FAQ

## **Q. When and where is EuroPython 2025 taking place?**

> A. EuroPython 2025 will take place on **14–20 July 2025**. Mark your calendars!

Reviewer 2: @cybit would you like to replace this with en-dash too?

Author: I think I applied it now in https://github.com/EuroPython/website/pull/949/commits/f66b84355d92b319c68fb692fbfc1742cd46e515 <3

> A. EuroPython 2025 will take place on **14–20 July 2025**. Mark your calendars!

Reviewer: ```suggestion
A. EuroPython 2025 will take place on **14 – 20 July 2025**. Mark your calendars!
```

## **Q. What will the schedule look like?**

A. The conference will be organised into three phases:

1. **Monday & Tuesday (14 & 15 July):** Tutorials & Workshops
2. **Wednesday–Friday (16–18 July):** Main Conference Days
3. **Saturday & Sunday (19 & 20 July):** Sprint Days

## **Q. What will the programme include?**

A. Expect a diverse programme with around **20 hands-on tutorials & workshops**, **120 talks**, and interactive discussions and events.

We aim to cover the many ways Python is used to solve problems and create exciting projects.

Curious about what to expect? Check out the [EP2024 programme](https://ep2024.europython.eu/schedule/) for inspiration.

We are always keen to hear new ideas on how to enrich the programme and other activities you are interested in oraganising.
Share your ideas with the programme team at[programme@europython.eu](mailto:programme@europython.eu).

## **Q. When can I submit a talk proposal?**

A. The [Call for Proposals (CfP)](/programme/cfp) is open from **10–27 January 2025**.

Need support with your submission? Explore our [Speaker Mentorship Programme](programme/mentorship/) for guidance.

## **Q. How are talks reviewed and selected?**

> A. This year, the first **100 proposals** will be quickly screened by the programme team, and any feedback for possible improvements will be provided.

Reviewer: ```suggestion
A. This year, the first **100 proposals** will be quickly screened by the programme team, and feedback for possible improvements will be provided.
```

Once the CfP closes, all proposals go through **community voting and two rounds of review and refinement** to finalise the programme.

Learn more about our selection process [here](/programme/selection/).

## **Q. What support is available for speakers?**
{/*
A. Accepted speakers receive a **free ticket**. Details about ticket for speakers will be shared later.
*/}

Speakers needing travel support are encouraged to apply for Financial Aid when applications open.

We actively support first-time speakers and welcome participation from under-represented groups in tech. Learn more through our [Speaker Mentorship Programme](/programme/mentorship).

Got any questions? Reach out to us at [programme@europython.eu](mailto:programme@europython.eu).

## **Q. When do ticket sales start, and how much will they cost?**

> A. Ticket sales will open soon! Stay tuned for updates.

Reviewer: ```suggestion
A. Ticket sales are expected to open in March. Stay tuned for updates!
```
